examples/catalog-module/index.js

```diff
@@ -1,13 +1,20 @@
-import {render} from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Catalog} from '../../src/index';
 
-render({
-  title: 'My Components',
-  pages: [
-    {
-      path: '/',
-      title: 'Foo',
-      imports: {Foo: require('./components/Foo/Foo')},
-      component: require('./components/Foo/Foo.docs.md')
-    }
-  ]
-}, document.getElementById('app'));
+ReactDOM.render(
+  <Catalog
+    configuration={{
```

Comment:
`Catalog.render()` still works the same way.

I wouldn't go with the pages as React elements approach. It would add too much complexity:

- We'd need to convert the React elements to a plain config anyway internally
- (while not actually rendering them)
- What if someone would mix the `pages` prop with pages as children? Which take precedence? FWIW, this is already [an issue](https://github.com/reactjs/react-router/issues/2306#issuecomment-149124993) in React Router (`childRoutes` on the plain route configuration which is also allowed as a prop on `<Route>` which in turn can contain nested routes).

I don't think it would be worth the hassle to do this. On the other hand, I'm in favor of declaring all top-level configuration as props on `<Catalog>`, e.g.

```js
<Catalog
  title='My Style Guide'
  theme={{}}
  basePath='/docs'
  imports={{}}
  pages={[
    {title: 'Introduction', path: '/', src: 'intro.md'}
  ]}
  /* etc. */
/>
```